### DISCUSSION TITLE
CPU temperature and GPU temperature are not displayed

### DISCUSSION THREAD

**Discussion by hx7697** [zhongyang219/TrafficMonitor#1791] [Q&A]
My CPU temperature and GPU temperature are not displayed, only show "--". My CPU is i7-13700k, GPU 6600XT

**Comment by 2915017823** [comment]
我也，如何解决

**Comment by NofuNagy** [comment]
You need to update librehardwaremonitorlib.dll in your TrafficMonitor Folder. The one that comes with TM is outdated.
Download the latest libre package: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/releases
Unzip it and copy the dll to your TM folder.
